### DISCUSSION TITLE
6x9?

### DISCUSSION THREAD

**Discussion by Morrisman68** [edgarkech/scalable_field_camera#18] [Ideas]
I wonder if anyone has tried, or even considered, scaling the design down to 6x9 format?

Thanks,

Ben

**Comment by edgarkech** [comment]
interesting consideration, but...

... there may be some challenges or restrictions:

- my design has some serious limitations when it comes to using wide angle lenses - for the 4x5 version 90mm is the shortest usable lens, for a scaled down version it wouldn't be much shorter, 75mm at best.
- Also, for a more compact camera I'd suggest changing the lens board format to something smaller - maybe Horseman style 80x80mm lens boards instead of the Linhof style lens boards 97x99mm
- for 6x9 there are no standard cassettes, and we would have to make modifications to the back for using 6x9 roll film backs (Graflex, Horseman, Linhof with Graflex clamps) 

all in all doable, but not with a simple scale down. I'd rather suggest using the brilliant MP120TC camera design from Kent Williams (https://williamskg6.wixsite.com/mp120tc or https://github.com/MP120TC/Current_Version).
